benchmark-service/benchmark/templates/index.html

```diff
@@ -2,11 +2,15 @@
 {% block title %} Benchmarks {% endblock %}
 
 {% block content %}
+<link rel="stylesheet" href="{{ base_path }}/static/css/autocomplete.css" />
+
 <h1>Benchmark System</h1>
 
-<form action="{{ base_path }}/" method="get">
-    <input type="text" name="file"></input>
-    <input type="submit" value="lookup"></input>
+<form autocomplete="off" action="{{ base_path }}/" method="get">
```

Comment:
Can you confirm whether this makes a new GET request each time you type a character? We don't want that behavior.

Reply:
???

Reply:
Wasn't sure if I was checking it correctly yesterday, but the browser shows GET requests being made only when you press the lookup button, not while typing